### PR TITLE
Subscribe/Unsubscribe

### PR DESCRIPTION
**Pull request by elGatoMantocko**
### Things I did
- added subscribe/unsubscribe button
- split the channels subscribed to into their own list
- clean up the home template a little
- use the set call to change channel subscriptions
- test cases for subscribe feature

### CODE REVIEW

File: client/controllers/public/home.js [hunk 1]
```diff
@@ -1,9 +1,59 @@
 Template.home.onCreated(function() {
-  Meteor.subscribe('latestChannels');
+  var self = this;
+  self.autorun(function() {
+    Meteor.subscribe('latestChannels');
+    if (Meteor.userId()) {
+      Meteor.subscribe('myChannels');
+    }
```
Comment (elGatoMantocko) [inline client/controllers/public/home.js]: I wanted to try to use a template instance variable here for the user, but there is a consensus that Meteor is anti-dry in this respect.

https://forums.meteor.com/t/access-parent-template-reactivevar-from-child-template/973/15

Comment (cjsauer) [inline client/controllers/public/home.js]: What would be the benefit of using a template variable? I'm not sure I understand the context.

**Comment by elGatoMantocko** [comment]
The test fails because `this.save()` returns 0 in the `Users.unsubscribeFrom(channel)` method.

If we can figure out why saving the user after removing the channel, then the tests will pass.

**Comment by marti584** [comment]
So, just to document this our decisions. This pull request is getting a little too dependent on #15 and it's getting larger than its intended method. So it might be better to nix this pull request and issue a new one after we accept #15 into the develop branch.

**Pull request closed without merging**